webapp/test/test_repos.py

```diff
@@ -1,14 +1,18 @@
 """Unit test for CveAPI module."""
-from test import schemas
+import datetime
+
+from test import schemas, tools
 from test.conftest import TestBase
 
 import pytest
+import pytz
 
 from repos import RepoAPI
 
 REPO_JSON_EMPTY = {}
 REPO_JSON_BAD = {"page_size": 9}
 REPO_JSON = {"repository_list": ["rhel-7-server-rpms"]}
+REPO_JSON_MODIFIED_SINCE = {"repository_list": ["rhel-7-server-rpms"], "modified_since": "2014-04-06T01:23:45+02:00"}
```

Comment:
You set some `"modified_since"` value here but you then change it in the test. Why? Isn't it easier directly to set it to value you need for that test?

Reply:
It sets an actual date, so whenever tests would be run it always will be the actual date

Reply:
Yes, but what is it good for? Why not simply set today timestamp there?

Reply:
:thinking: maybe, you right, I just made in a similar way how it was in errata and cve tests

Reply:
Yes, I see it. It seems to be originally created by @psegedy. Guys @tkasparek @jdobes @eherget, what do you think about it? I think it would be better to rewrite it everywhere to constant timestamp instead of datetime.now(). It's better to have deterministic tests.

Reply:
+1 to that, but I'd fix this just for this PR and would fix the rest in another PR.

Reply:
@tkasparek thanks, I agree.

Reply:
Ok, I'll do it